README.md

```diff
@@ -195,6 +196,61 @@ $ oc start-build example-quarkus-binary --from-dir=target
 And that's it. In a couple minutes you should have your Kogito Service application up and running using the same binaries
 built locally.
 
+### Build From File
+
+Kogito Operator can configure a build for you that will pull and build your application through the source to image (s2i)
+feature (available only on OpenShift). This kind of build consists id upload a Kogito service file, e.g. a dmn or a bpmn file
+to the OpenShift Cluster and trigger a new Source to Image build automatically.
+
+You can provide a single dmn, drl or bpmn|bpmn2 files, the standalone file can be locally on the filesystem or on a Web Site, like Git Hub:
+
+
+```bash
+# from filesystem
+$ kogito -service example-dmn-quarkus --from-file TrafficViolation.dmn
+File found /tmp/kogito-test/src/main/resources/TrafficViolation.dmn.
+...
+The requested file(s) was successfully upload to OpenShift, a build with this file should now be running. To see its logs, run 'oc logs -f bc/test-builder -n kogito'
+```
+
+While checking OpenShift build logs with this command **oc logs -f bc/test-builder -n kogito** you can see on the beginning ot the build log:
+```
+Receiving source from STDIN as file TrafficViolation.dmn
+Using docker-registry.default.svc:5000/spolti-1/kogito-quarkus-ubi8-s2i@sha256:729e158710dedba50a49943ba188d8f31d09568634896de9b903838fc4e34e94 as the s2i builder image
+```
+
+```bash
+# from a git repository
+$ kogito -service example-dmn-quarkus --from-file https://raw.githubusercontent.com/kiegroup/kogito-examples/master/dmn-quarkus-example/src/main/resources/Traffic%20Violation.dmn
+...
+The requested file(s) was successfully upload to OpenShift, a build with this file should now be running. To see its logs, run 'oc logs -f bc/test-builder -n kogito'
+```
+
+If you check the logs, we can also see the dmn file being used to to build the Kogito service.
+
+Sometimes we also need to provide more than one Kogito resource file, e.g. a drl and a bpmn. In that case all we need to do
+is, instead specify a directory instead, if any valid file is found on the given directory, the cli will compress these
```

Comment:
> if any valid file is found on the given directory

So, only files with specific suffix are copied?

Reply:
yes, IMHO there's no point to allow user to push any kind of file since the archetypes can not work properly.

Reply:
does it make sense?

Reply:
it can be improved in the future if needed.
README updated.